### ISSUE TITLE
Install error

### ISSUE THREAD

**Issue by robisoft** [AsoboStudio/glTF-Blender-IO-MSFS#285]
### Current Behavior

When I try to activate the plugin I obtain this error:
ModuleNotFoundError: No module named 'io_scene_gltf2.blender.exp.gltf2_blender_gather_texture_info'

### Expected Behavior

<img width="641" alt="image" src="https://github.com/AsoboStudio/glTF-Blender-IO-MSFS/assets/778386/8d604653-7203-4841-a275-d781d8a8b7f3">

### Steps To Reproduce

1. download plugin zip file 1.3.2 version
2. Install it (edit,preferences,plugins - install)
3. activate it (here the error appears)

### Environment

```markdown
- OS: Windows 11
- Blender: 3.6.5
- glTF-Blender-IO-MSFS: 1.3.2
- python: 3.12.0
```

**Comment by ronh991** [comment]
Does not work in 3.6 - as per the release notes.

**Comment by SirComand** [comment]
yes it works in 3.6.1 but you have to copy the gltf addon from 3.3 to the blender install root and replace it.

**Comment by ronh991** [comment]
> yes it works in 3.6.1 but you have to copy the gltf addon from 3.3 to the blender install root and replace it.

Yes, OR ....... you can change one line in the \io_scene_gltf2_msfs\io\msfs_material.py file

from this

from io_scene_gltf2.blender.exp.gltf2_blender_gather_texture_info import (

to this

from io_scene_gltf2.blender.exp.material.gltf2_blender_gather_texture_info import (

line 16 add in .material

But you should be aware of the Blender node name change of ShaderNodeMixRGB to ShaderNodeMix.  This also means the input numbering has changed also.  If you get any errors exporting your blend file it could be due to Blender 3.6 changes

**Comment by y-khodja** [comment]
Hello,

We don't support the plugin for Blender 3.6.x, it's available for Blender 3.3.x only for now.

Thank you for your report.
